docs/installation-guide.md

```diff
@@ -40,42 +40,46 @@ followed by :
 
 ### Procedure
 
-* [Build the CarbonData](https://cwiki.apache.org/confluence/display/CARBONDATA/Building+CarbonData+And+IDE+Configuration) project and get the assembly jar from "./assembly/target/scala-2.10/carbondata_xxx.jar" and put in the ``"<SPARK_HOME>/carbonlib"`` folder.
+1. [Build the CarbonData](https://cwiki.apache.org/confluence/display/CARBONDATA/Building+CarbonData+And+IDE+Configuration) project and get the assembly jar from `./assembly/target/scala-2.1x/carbondata_xxx.jar`. 
```

Comment:
Please use "https://github.com/apache/incubator-carbondata/blob/master/build/README.md" replace "https://cwiki.apache.org/confluence/display/CARBONDATA/Building+CarbonData+And+IDE+Configuration"